src/BlockEntities/DropSpenserEntity.cpp

```diff
@@ -75,7 +75,7 @@ void cDropSpenserEntity::DropSpense(cChunk & a_Chunk)
 	if (SlotsCnt == 0)
 	{
 		// Nothing in the dropspenser, play the click sound
-		m_World->BroadcastSoundEffect("block.dispenser.fail", static_cast<double>(m_PosX), static_cast<double>(m_PosY), static_cast<double>(m_PosZ), 1.0f, 1.2f);
+		m_World->BroadcastSoundEffect("block.dispenser.fail", {static_cast<double>(m_PosX), static_cast<double>(m_PosY), static_cast<double>(m_PosZ)}, 1.0f, 1.2f);
```

Comment:
Would `Vector3d(m_PosX, m_PosY, m_PosZ)`  work instead?

Reply:
I think it should, and it's much prettier, yes :)

Reply:
Anyway, it's temporary, because `m_Pos` should be rewritten into a vector eventually, and conversions between the vector types are defined.